IntegrationTests/LayoutEventsTest.js

```diff
@@ -36,56 +35,80 @@ type State = {
   containerStyle?: ViewStyleProp,
 };
 
-const LayoutEventsTest = createReactClass({
-  displayName: 'LayoutEventsTest',
-  getInitialState(): State {
-    return {
-      didAnimation: false,
-    };
-  },
-  animateViewLayout: function() {
+class LayoutEventsTest extends React.Component<$FlowFixMeProps, State> {
+  _view: ?View;
+  _img: ?Image;
+  _txt: ?Text;
+  _isMounted: boolean = true;
```

Comment:
Why are you setting `_isMounted` to true initially, instead of doing this:

```JavaScript
  _isMounted: boolean = false;
  componentDidMount() {
    this._isMounted = true;
  }
```

Reply:
Because... there's no reason, it's a mistake !